Dockerfile

```
@@ -1,205 +1,62 @@
-    # -------------
-    # RUN adduser --disabled-password --gecos "" appuser && chown -R appuser /app
-    # USER appuser
-    
-    # -------------
-    # Final start command
-    # -------------
-    CMD ["/app/start.sh"]
+FROM elixir:1.15-slim AS builder
+
+# Install build dependencies
+RUN apt-get update -y && \
```

Hadolint (2.12.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)